addons/pos_restaurant/models/pos_restaurant.py

```diff
@@ -20,15 +20,14 @@ class RestaurantFloor(models.Model):
     sequence = fields.Integer('Sequence', default=1)
     active = fields.Boolean(default=True)
     floor_background_image = fields.Image(string='Floor Background Image')
-    floor_prefix = fields.Integer('Floor Prefix', default=1, help="The prefix will be used when creating a new table in the PoS interface.")
```

Comment:
Cannot be removed in stable. Do it in master. And an upgrade script will be needed